Crashlytics/CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 
+# v4.0.0-beta.6
+
+ - [added] Added a new API to record custom exception models and stacktraces to Crashlytics. This is a replacement for the `recordCustomException` API that existed in the Fabric Crashlytics SDK (#5055)
+ - [fixed] Fixed an issue with the `sendUnsentReports` API where reports wouldn't be uploaded until the method was called twice in specific instances (#5060)
+ - [changed] Changed the origin that Crashlytics uses to register Crash events for Crash Free Users. Ensure you have installed Firebase Analytics version x.x.x or above (#5030)
```

Comment:
@jasonhu-g do you know what version is needed for FirebaseAnalytics?

Reply:
The latest which is:
https://github.com/CocoaPods/Specs/blob/master/Specs/e/2/1/FirebaseAnalytics/6.3.1/FirebaseAnalytics.podspec.json